doc/source/usage.rst

```diff
@@ -10,18 +10,29 @@ localhost:
 
 .. code-block:: python
 
-    >>> from pylxd.client import Client
+    >>> from pylxd import Client
     >>> client = Client()
 
+If your LXD instance is listening on HTTPS, you can pass a two part tuple
+of (cert, key) as the `cert` argument.
+
+.. code-block:: python
+
+    >>> from pylxd import Client
+    >>> client = Client(
+    ...     endpoint='http://10.0.0.1:8443',
+    ...     cert=('/path/to/client.crt', '/path/to/client.key'))
+
+Note: in the case where the certificate is self signed (LXD default),
+you may need to pass `verify=False`.
+
 This :class:`~client.Client` object exposes managers for:
 
 - :class:`~container.Container`,
 - :class:`~profile.Profile`,
 - :class:`~operation.Operation`,
 - :class:`~image.Image`,
 
-Also, it exposes the HTTP API with the `api <api.html#Client.api>`_ attribute,
-allowing lower-level operations.
```

Comment:
Hi @jpic - the operations we can't do with Client (or the other APIs) should be implemented. The `api` property's existence is a byproduct of the architecture, but I don't want to encourage its use.